package/network/config/netifd/files/usr/libexec/network/packet-steering.sh

```diff
@@ -25,46 +26,50 @@ find_irq_cpu() {
 }
 
 set_hex_val() {
-	local file="$1"
-	local val="$2"
+	file="$1"
+	val="$2"
 	val="$(printf %x "$val")"
-	[ -n "$DEBUG" ] && echo "$file = $val"
+    [ -n "$DEBUG" ] && echo "$file = $val"
 	echo "$val" > "$file"
 }
 
 packet_steering="$(uci get "network.@globals[0].packet_steering")"
 [ "$packet_steering" != 1 ] && exit 0
 
-exec 512>/var/lock/smp_tune.lock
-flock 512 || exit 1
+exec 9>/var/lock/smp_tune.lock
+flock 9 || exit 1
 
 [ -e "/usr/libexec/platform/packet-steering.sh" ] && {
-	/usr/libexec/platform/packet-steering.sh
+    /usr/libexec/platform/packet-steering.sh
 	exit 0
 }
 
 for dev in /sys/class/net/*; do
-	[ -d "$dev" ] || continue
+    [ -d "$dev" ] || continue
 
-	# ignore virtual interfaces
-	[ -n "$(ls "${dev}/" | grep '^lower_')" ] && continue
+    # ignore virtual interfaces
+    for file in "${dev}"/lower_*; do
```

Comment:
makes more sense to endquote * rather than dev

Reply:
Fixed